pkg/allocator/allocator.go

```diff
@@ -676,11 +696,20 @@ func (a *Allocator) Allocate(ctx context.Context, key AllocatorKey) (idpool.ID,
 
 	kvstore.Trace("Allocating from kvstore", nil, logrus.Fields{fieldKey: key})
```

Comment:
nit: this log line should be below, since we're not allocating from kvstore when operatorIDManagement is enabled :-)

Reply:
Done.